src/GlobalScreen/Scope/Layout/Provider/PagePart/StandardPagePartProvider.php

```diff
@@ -269,10 +270,13 @@ public function getViewTitle(): string
     /**
      * @inheritDoc
      */
-    public function getToastContainer(): TContainer
+    public function getToastContainer(): ?TContainer
     {
-        $toast_container = $this->ui->factory()->toast()->container();
+        if ((new ilSetting('notifications'))->get('enable_osd') !== '1') {
```

Comment:
@iszmais I should have stayed firm about the scope Ttoasts`, this shows that they definitely belong to the `Notifications` :-)

See my comment to #5698. Here it is a little less tragic with the ilXY classes, there are (unfortunately) already others. However, I think that we should provide the settings relevant for the GS externally when initialising the service in order to abstract this cleanly. 

I will make you a proposal for this shortly.